neo/rawio/baserawio.py

```diff
@@ -735,7 +769,7 @@ def setup_cache(self, cache_path, **init_kargs):
             dirname = os.path.dirname(resource_name)
         else:
             assert os.path.exists(cache_path), \
-                'cache_path do not exists use "home" or "same_as_resource" to make this auto'
+                'cache_path does not exists use "home" or "same_as_resource" to make this auto'
```

Comment:
Not really related to the PR, but I don't understand what this error message is telling me. More punctutation and context needed.

Reply:
Neither do I. But I think @samuelgarcia would have to comment on that.